debian/vyatta-dataplane-flowstat-plugin.install

```diff
@@ -0,0 +1,5 @@
+usr/lib/*/vyatta-dataplane/pipeline/plugins/flowstat_plugin.so
+usr/share/flowstat_plugin/protobuf/* /usr/share/vyatta-dataplane/protobuf
+usr/share/perl5/flowstat_plugin/proto/* /usr/share/perl5/vyatta/proto
+src/pipeline/nodes/flowstat/yang/*.yang usr/share/configd/yang
```

Comment:
The yang file should be in it's own debian package named for the yang module.

Similarly the dataplane plugin should be separate debian package from the control plane daemon/scripts, with appropriate dependencies between them.

Reply:
Thanks, I will split into some repositories.